sdk/ai/azure-ai-projects/README.md

```diff
@@ -571,6 +571,108 @@ print(f"Created agent, agent ID: {agent.id}")
+
+Now we will create the function itself. Install [.NET](https://dotnet.microsoft.com/download) and [Core Tools](https://go.microsoft.com/fwlink/?linkid=2174087) and create the function project using next commands. 
+```
+func init FunctionProj --worker-runtime dotnet-isolated --target-framework net8.0
+cd FunctionProj
+func new --name foo --template "HTTP trigger" --authlevel "anonymous"
+dotnet add package Azure.Identity
+dotnet add package Microsoft.Azure.Functions.Worker.Extensions.Storage.Queues --prerelease
+```
+
+**Note:** There is a "Azure Queue Storage trigger", however the attempt to use it results in error for now.
+We have created a project, containing HTTP-triggered azure function with the logic in `Foo.cs` file. As far as we need to trigger Azure function by a new message in the queue, we will replace the content of a Foo.cs by the C# sample code above. 
+To deploy the function run the command from dotnet project folder:
+
+```
+func azure functionapp publish function_name
+```
+
+In the `storage_account_already_present_in_resource_group` select the `Queue service` and create two queues: `azure-function-foo-input` and `azure-function-tool-output`. Note that the same queues are used in our python sample. To check that the function is working, place the next message into the `azure-function-foo-input` and replace `storage_account_already_present_in_resource_group` by the actual resource group name, or just copy the output queue address.
+```json
+{
+  "OutputQueueUri": "https://storage_account_already_present_in_resource_group.queue.core.windows.net/azure-function-tool-output",
+  "CorrelationId": "42"
+}
+```
+
+Next, we will monitor the output queue or the message. You should receive the next message.
+```json
+{
+  "Value": "Bar",
+  "CorrelationId": "42"
+}
+```
+Please note that the input `CorrelationId` is the same as output.
+*Hint:* Place multiple messages to input queue and keep second internet browser window with the output queue open and hit the refresh button on the portal user interface, so that you will not miss the message. If the message instead went to `azure-function-foo-input-poison` queue, the function completed with error, please check your setup.
+After we have tested the function and made sure it works, please make sure that the project have the next roles for the storage account: `Storage Account Contributor`, `Storage Blob Data Contributor`, `Storage File Data Privileged Contributor`, `Storage Queue Data Contributor` and `Storage Table Data Contributor`. Now the function is ready to be used by the agent.
```

Comment:
What do you mean by "the project" here? Can you give the exact Azure resource name?

Reply:
Fixed